docs/eclipse/release-notes.md

```diff
@@ -5,6 +5,15 @@ title: Release Notes
 
 Here you will find the latest information about releases of the jSparrow Eclipse plugin.
 
+## 3.14.0 <Badge vertical="middle" text="20.02.2020"/>
+
+The [70<sup>th</sup> refactoring rule](/rules/) is shipped with jSparrow 3.14.0. It improves your `java.util.Optional`.
+
+### New Rule
+#### [**Use Optional::filter**](/rules/optional-filter.html)
+
+This rule extracts an `Optional::filter` from the consumer used in `Optional::ifPresent`. This makes the lambda expression easier to read and it can be combined with other `Optional` operations.
```

Comment:
This second sentence doesn't sound good to me. How about replacing it with: 
 "This simplifies the lambda expression used with Optional operations."

(It is also used in the description of the frontmatter)